modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/DisabledArea.scss

```diff
@@ -1,37 +1,39 @@
-@import 'atlas-variables';
+@import '_cadmin-variables';
 
 @import './variables';
 
-.page-editor__disabled-area {
-	cursor: not-allowed;
-	position: relative;
-	transition: padding-left ease $defaultTransitionDuration,
-		padding-right ease $defaultTransitionDuration;
-	will-change: padding-left, padding-right;
+html#{$cadmin-selector} {
```

Comment:
Could this be done through a controlled mixin instead of having to manually set the selector like this?

```scss
scope({
...
})

// or 
isolate({
...
})
```

Reply:
Yeah we could, it would look like:
```
@include isolate() {
   .cadmin {
        .page-editor__sidebar__buttons {}
    }

    body.has-control-mnenu .cadmin.page-editor__sidebar {}
}
```
The benefit is it's easier to remember and less typos?